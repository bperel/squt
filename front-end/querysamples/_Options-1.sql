SELECT DISTINCT SQL_SMALL_RESULT SQL_BUFFER_RESULT SQL_CACHE SQL_CALC_FOUND_ROWS HIGH_PRIORITY STRAIGHT_JOIN b.a
FROM b
FOR UPDATE;