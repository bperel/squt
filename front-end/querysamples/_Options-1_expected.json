{
	"Tables" : {
		"b" : {
			"b" : {
				"OUTPUT" : {
					"a" : {
						"-1" : "a"
					}
				}
			}
		}
	},
	"Options" : {
		"SELECT_DISTINCT" : 1,
		"TL_WRITE" : 1,
		"OPTION_FOUND_ROWS" : 1,
		"OPTION_TO_QUERY_CACHE" : 1,
		"SELECT_STRAIGHT_JOIN" : 1,
		"TL_READ_HIGH_PRIORITY" : 1,
		"OPTION_BUFFER_RESULT" : 1,
		"SQL_NO_CACHE" : 1,
		"SELECT_SMALL_RESULT" : 1
	}
}